src/Forms/GridFieldRefreshButton.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
```

Comment:
Can you please put this class into a namespace, to aid with upgrading to 4?

Reply:
This is what the [upgrader tool](https://github.com/silverstripe/silverstripe-upgrader) does, I don't see any benefit in doing it manually now, for the sake of saving ~two minutes in the upgrade task (#42). If we do it for one class, we should do it for all classes (the GridField components at least), which has relatively trivial yet rather time consuming knock-on affects (e.g. template resolving, etc).

Reply:
Fair enough. Do you want to revert the files we've already namespaced so far or at least make a decision as to whether to continue doing it where we can?

Reply:
I think what's done is done. What I meant by consistency is that it would be weird to have one namespaced gridfield component, but two others that aren't.
The util class is fine I think, being a util class.

After curiosity got the better of me in namespaces and gridfieldcomponents that hold routes in SS4 (which commonly rely on classname) I am about to include these namespaces after all 😅 (in another PR or course). But I didn't see it as necessary to get this one over the line :>